core/variable_map.js

```diff
@@ -358,9 +358,6 @@ VariableMap.prototype.getVariablesOfType = function(type) {
 VariableMap.prototype.getVariableTypes = function(ws) {
   const variableMap = {};
   object.mixin(variableMap, this.variableMap_);
-  if (ws && ws.getPotentialVariableMap()) {
-    object.mixin(variableMap, ws.getPotentialVariableMap().variableMap_);
-  }
```

Comment:
This eslint warning is a sign that in fact this code does need to be retained, but modified appropriately:
```JS
  if (ws) {
    Object.assign(variableMap, ws.getVariableMap().variableMap_);
  }
```